salt/cloud/clouds/dimensiondata.py

```diff
@@ -38,6 +38,7 @@
 
     HAS_LIBCLOUD = True
 except ImportError:
+    log.exception("You do not have apache-libcloud 0.20<= installed")
```

Comment:
Sure, but we should handle that in the module, rather than in that file. That's what the `__virtual__()` function can be used for. Similar to something like this: https://github.com/saltstack/salt/blob/develop/salt/modules/boto_lambda.py#L113